client/v3/concurrency/stm_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2016 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package concurrency
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGet(t *testing.T) {
+	tests := []struct {
+		name string
+		stm  *stmSerializable
+		in   []string
+		resp string
+	}{
+		{
+			name: "Empty keys returns empty string",
+			stm:  &stmSerializable{},
+			in:   []string{},
+			resp: "",
+		},
+		{
+			name: "Nil keys returns empty string",
+			stm:  &stmSerializable{},
+			in:   nil,
+			resp: "",
+		},
+	}
+
+	for _, test := range tests {
```

Comment:
please consider to use `test := test` in Line 45. Just in case that new test var will shadow the old one, when we enable parallel mode.

Reply:
This comment applies lots of other test cases as well.